### PR TITLE
Add further information to online-offline event documentation

### LINKED ISSUE #7525: online-offline-events tutorial is misleading
docs/tutorial/online-offline-events.md

The docs use the `navigator.onLine()` statement and events as an indicator if a browser is on/offline. This is not a good idea, as this will only tell you if a user's computer is connected to a local network - not if they have internet connectivity.

Electron users will be using this tutorial to create offline pages, which simply won't appear in a large percentage of cases where there is no connectivity. I know because we have had lots of user reports of a blank screen where nothing can be clicked :-)

Either the tutorial article should be altered to be shown offline when the `window.loadURL` statement fails, the tutorial removed, or have a big red warning at the top saying that it will be useless in a good chunk of cases.

### PR DESCRIPTION
**Pull request by nabeelomer**
Fixes #7525

### CODE REVIEW

File: docs/tutorial/online-offline-events.md [hunk 1]
```diff
@@ -78,3 +78,4 @@ _online-status.html_
 </body>
 </html>
 ```
+In Chrome and Safari, if the browser is not able to connect to a local area network (LAN) or a router, it is offline; all other conditions return true. So while you can assume that the browser is offline when it returns a false value, you cannot assume that a true value necessarily means that the browser can access the internet. You could be getting false positives, such as in cases where the computer is running a virtualization software that has virtual ethernet adapters that are always "connected." Therefore, if you really want to determine the online status of the browser, you should develop additional means for checking.
```
Comment (MarshallOfSound) [inline docs/tutorial/online-offline-events.md]: Just copy pasting the paragraph (although mostly correct) doesn't quite solve the issue.

There is no point referring to "browsers" or "chrome" or "safari" as these concepts to not relate to Electron.  Also to make markdown more readable can you cut each line off at around 80 characters (most IDE's have a line there).

Should end up something like

```markdown
**NOTE:** If Electron is not able to connect to a local area network (LAN) or a router, it is considered offline; all other conditions return true. So while you can assume that the browser is offline when `navigator.onLine` returns a false value, you cannot assume that a true value necessarily means that the browser can access the internet. You could be getting false positives, such as in cases where the computer is running a virtualization software that has virtual ethernet adapters that are always "connected." Therefore, if you really want to determine the online status of the browser, you should develop additional means for checking.
```

Comment (nabeelomer) [inline docs/tutorial/online-offline-events.md]: Thank you, I made the change.

File: docs/tutorial/online-offline-events.md [hunk 2]
```diff
@@ -78,3 +78,12 @@ _online-status.html_
...
 </html>
 ```
+**NOTE:** If Electron is not able to connect to a local area network (LAN) or
+a router, it is considered offline; all other conditions return true.
+So while you can assume that the browser is offline when `navigator.onLine`
+returns a false value, you cannot assume that a true value necessarily
+means that the browser can access the internet. You could be getting false
+positives, such as in cases where the computer is running a virtualization
+software that has virtual ethernet adapters that are always "connected."
+Therefore, if you really want to determine the online status of the browser,
```
Comment (MarshallOfSound) [inline docs/tutorial/online-offline-events.md]: Still says "browser" here, should change to 'Electron'

**Comment by kevinsawicki** [comment]
Thanks for this 👍 🚢

**Pull request merged**